api/v1alpha1/runnerpool_types.go

```diff
@@ -73,6 +73,11 @@ type RunnerPodTemplateSec struct {
 	// List of volumes that can be mounted by containers belonging to the pod.
 	// +optional
 	Volumes []corev1.Volume `json:"volumes,omitempty"`
+
+	// Name of the service account that the Pod use.
+	// +kubebuilder:default="default"
```

Comment:
Please confirm the default value is set.

https://github.com/cybozu-go/github-actions-controller/blob/bc31945544bd31580993f9ea1fe81df951671657/api/v1alpha1/runnerpool_webhook_test.go#L65-L68